src/compiler/objective_c_plugin.cc

```diff
@@ -52,15 +52,16 @@ inline ::grpc::string ImportProtoHeaders(
 
   ::grpc::string base_name = header;
   grpc_generator::StripPrefix(&base_name, "google/protobuf/");
```

Comment:
I am thinking that if protobuf moved things out of `google/protobuf`, users should probably not continue to use `google/protobuf` prefix in their header? That's a breaking change though so I'm not sure if that's true.

Reply:
They only changed paths of **.pbobjc.h/m** files.  For **.proto** files, their paths are still the same. The `header` variable is generated based these **.proto** files, so we still neet to strip `google/protobuf `prefix.

Reply:
Ah that makes sense!